src/components/create/PegInAccountSelect.vue

```diff
@@ -137,26 +137,41 @@ export default class PegInAccountSelect extends Vue {
     });
   }
 
+  @Emit()
+  isAccountEnabled(accountType:BtcAccount): boolean {
+    let enabled:boolean;
+    switch (accountType) {
+      case constants.BITCOIN_SEGWIT_ADDRESS:
+        enabled = this.pegInTxState.bitcoinWallet !== constants.WALLET_LIQUALITY;
+        break;
+      case constants.BITCOIN_LEGACY_ADDRESS:
+        enabled = this.pegInTxState.bitcoinWallet !== constants.WALLET_LIQUALITY;
```

Comment:
The line inside case constants.BITCOIN_SEGWIT_ADDRESS: and case constants.BITCOIN_LEGACY_ADDRESS: is exactly the same:

enabled = this.pegInTxState.bitcoinWallet !== constants.WALLET_LIQUALITY;

Why don't we join them both?

switch (accountType) {
      case constants.BITCOIN_SEGWIT_ADDRESS:
      case constants.BITCOIN_LEGACY_ADDRESS:
        enabled = this.pegInTxState.bitcoinWallet !== constants.WALLET_LIQUALITY;
        break;
      default:
        enabled = true;
    }